src/transformers/models/clip/tokenization_clip_fast.py

```diff
@@ -124,44 +92,81 @@ def __init__(
             bos_token=bos_token,
             eos_token=eos_token,
             pad_token=pad_token,
-            add_prefix_space=add_prefix_space,
             **kwargs,
         )
 
-        pre_tok_state = json.loads(self.backend_tokenizer.pre_tokenizer.__getstate__())
-        if pre_tok_state.get("add_prefix_space", add_prefix_space) != add_prefix_space:
-            pre_tok_class = getattr(pre_tokenizers, pre_tok_state.pop("type"))
-            pre_tok_state["add_prefix_space"] = add_prefix_space
-            self.backend_tokenizer.pre_tokenizer = pre_tok_class(**pre_tok_state)
+        if not isinstance(self.backend_tokenizer.pre_tokenizer, pre_tokenizers.Sequence):
+            raise ValueError(
+                "The backend_tokenizer provided does not match the expected format. The CLIP tokenizer has been "
+                "heavily modified from transformers version 4.17.0. If you want to use the new fast version of the "
+                "CLIP tokenizer, the easiest way for you is probably to do: "
+                '`CLIPTokenizerFast.from_pretrained("Path to local folder or Hub repo with slow tokenizer files", from_slow=True)`.'
+                " If you want to use your existing backend_tokenizer, you will have to download a version prior to "
+                "4.17.0 of transformers."
+            )
```

Comment:
Here's my addition to raise an error if the user try to import an old `tokenizer.json` file with the new `ClipTokenizerFast`